### PR TITLE
Add some filters to beneficiary list and clear wrong indentations

### PR DESCRIPTION
**Pull request by tirgil**
Added filters are : human_resource_id, location_id,marital_status, registered, enrolled_in_school

### CODE REVIEW

File: modules/templates/STL/config.py [hunk 1]
```diff
@@ -1859,31 +1860,55 @@ def custom_prep(r):
                                             #label = T("Branch"),
                                             options = s3_get_filter_opts("org_organisation"),
                                             hidden = True,
-                                            ),
+                                           ),
```
Comment (nursix) [inline modules/templates/STL/config.py]: Closing parenthesis should be left-aligned with the parameter list - so you shouldn't remove this space ;)

File: modules/templates/STL/config.py [hunk 2]
```diff
@@ -1859,31 +1860,55 @@ def custom_prep(r):
...
+                                            cols = 3,                           
+                                            hidden = True,
+                                            options = {True: T("Yes"),
+                                                       False: T("No"),
+                                                      },
+                                           ),
+                            S3OptionsFilter("case_details.enrolled_in_school",
+                                            cols = 3,
+                                            hidden = True,
+                                            options = {True: T("Yes"),
```
Comment (nursix) [inline modules/templates/STL/config.py]: You're repeating this True|False options dict - better to make it a local variable and re-use.

File: modules/templates/STL/config.py [hunk 3]
```diff
@@ -1859,31 +1860,55 @@ def custom_prep(r):
...
+                                           ),
+                            S3OptionsFilter("case_details.registered",
+                                            cols = 3,                           
+                                            hidden = True,
+                                            options = {True: T("Yes"),
+                                                       False: T("No"),
+                                                      },
+                                           ),
+                            S3OptionsFilter("case_details.enrolled_in_school",
+                                            cols = 3,
```
Comment (nursix) [inline modules/templates/STL/config.py]: What's the 3rd column with two options?

File: modules/templates/STL/config.py [hunk 4]
```diff
@@ -1859,31 +1860,55 @@ def custom_prep(r):
...
-                                              ),
+                                             ),
                             S3OptionsFilter("dvr_case_activity.project_id",
                                             options = s3_get_filter_opts("project_project"),
                                             hidden = True,
+                                           ),
+                            S3OptionsFilter("dvr_case.human_resource_id",
+                                            hidden = True,
+                                           ),
+                            S3LocationFilter("location_id",
```
Comment (nursix) [inline modules/templates/STL/config.py]: If this is after the beneficiary address, then it shouldn't be "location_id", but "address.location_id".

File: modules/templates/STL/config.py (hunk 1 above)
Comment (tirgil) [inline modules/templates/STL/config.py]: My misunderstood. @nursix done.

File: modules/templates/STL/config.py (hunk 4 above)
Comment (tirgil) [inline modules/templates/STL/config.py]: @nursix done

File: modules/templates/STL/config.py (hunk 3 above)
Comment (tirgil) [inline modules/templates/STL/config.py]: I just copy it from closed filter (dvr_case.status_id) that i thought it must be one more plus from count. I changed it to 2. @nursix done.

File: modules/templates/STL/config.py (hunk 2 above)
Comment (tirgil) [inline modules/templates/STL/config.py]: @nursix done.

**Pull request merged**